web/app/components/recipeCard/recipeCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, Text, Button } from "@mantine/core"
+import ImgFiller from "../../../public/Image Item.jpg"
```

Comment:
Rather than importing this we should just use the relative URL in the src attribute